Solutions/VCenter/Data Connectors/Connector_Syslog_vcenter.json

```diff
@@ -0,0 +1,138 @@
+{ 
+    "id": "vCenter",
+    "title": "VMware vCenter solution", 
+    "publisher": "VMware ", 
+    "descriptionMarkdown": "The [vCenter](https://www.vmware.com/in/products/vcenter-server.html) connector allows you to easily connect your vCenter server logs with Microsoft Sentinel. This gives you more insight into your organization's data centers and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **vCenter** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-vcenter-parser)",
+    "graphQueries": [ 
+     { 
+         "metricName": "Total data received", 
+         "legend": "vCenter", 
+         "baseQuery": "vCenter" 
+     } 
+    ], 
+          "sampleQueries": [ 
+           { 
+               "description" : "Total Events by Event Type", 
+               "query": "vCenter \n | summarize count() by eventtype"    
+           },
+        {
+            "description" : "log in/out to vCenter Server",
+            "query": "vCenter \n | where eventtype in ('UserLogoutSessionEvent','UserLoginSessionEvent') \n | summarize count() by tostring(eventtype),tostring(eventid),tostring(username),tostring(useragent) \n | top 10 by count_"  
```

Comment:
Based on the suggested modification in the Parser, tostring() will not be required here.

Reply:
change completed